Add ${PREFIX}/etc as an acceptable location for emboss.default.  (I'm still
waiting for this to be incorporated into an official release of EMBOSS.)

The port installs ${PREFIX}/etc/emboss.default.sample

-- Johann <johann@egenetics.com>


--- ajax/ajnam.c.orig	Thu Aug 17 20:31:30 2000
+++ ajax/ajnam.c	Mon Aug 28 15:12:09 2000
@@ -984,6 +984,15 @@
 	prefixRootFile = fopen (ajStrStr(namRootStr), "r");
     }
 
+    /* look for $(PREFIX)_ROOT/../emboss.default */
+
+    if (!prefixRootFile)
+    {
+      ajFmtPrintS (&namRootStr, "%s/etc/%s.default",
+		   namInstallRoot, prefix);
+      prefixRootFile = fopen (ajStrStr(namRootStr), "r");
+    }
+
     if (namFileOrig)
 	(void) ajStrAppC (&namFileOrig, ", ");
     (void) ajStrApp (&namFileOrig, namRootStr);
